context menu broken

* https://camptocamp.github.io/ngeo/master/examples/contribs/gmf/apps/desktop
 * right click

```
desktop.js:140 Error: [$parse:syntax] http://errors.angularjs.org/1.6.4/$parse/syntax?p0=%7B&p1=is%20an%20unexpec…n&p2=40&p3=coord_4326%7CngeoNumberCoordinates%3A2%3A%7By%7D%20%7Bx&p4=%7Bx
    at desktop.js:23
    at s.throwError (desktop.js:253)
    at s.ast (desktop.js:245)
    at uc.parse (desktop.js:280)
    at desktop.js:149
    at k (desktop.js:127)
    at la (desktop.js:98)
    at jc (desktop.js:82)
    at Ma (desktop.js:79)
    at Ma (desktop.js:79)
```

--- a/contribs/gmf/apps/desktop/contextualdata.html
+++ b/contribs/gmf/apps/desktop/contextualdata.html
@@ -12,7 +12,7 @@
       Wgs Coord.
     </td>
     <td>
-      {{coord_4326|ngeoNumberCoordinates:2:{y} {x}}
+      {{coord_4326|ngeoNumberCoordinates:2:'{y} {x}'}}
     </td>
   </tr>
   <tr>
@@ -20,7 +20,7 @@
       Wgs Coord. DMS
     </td>
     <td>
-      {{coord_4326|ngeoDMSCoordinates:0:{y} {x}}}
+      {{coord_4326|ngeoDMSCoordinates:0:'{y} {x}'}}
     </td>
   </tr>
   <tr>

--- a/contribs/gmf/apps/oeedit/contextualdata.html
+++ b/contribs/gmf/apps/oeedit/contextualdata.html
@@ -12,7 +12,7 @@
       Wgs Coord.
     </td>
     <td>
-      {{coord_4326|ngeoNumberCoordinates:2:{y} {x}}}
+      {{coord_4326|ngeoNumberCoordinates:2:'{y} {x}'}}
     </td>
   </tr>
   <tr>
@@ -20,7 +20,7 @@
       Wgs Coord. DMS
     </td>
     <td>
-      {{coord_4326|ngeoDMSCoordinates:0:{y} {x}}}
+      {{coord_4326|ngeoDMSCoordinates:0:'{y} {x}'}}
     </td>
   </tr>
   <tr>

--- a/contribs/gmf/apps/oeview/contextualdata.html
+++ b/contribs/gmf/apps/oeview/contextualdata.html
@@ -12,7 +12,7 @@
       Wgs Coord.
     </td>
     <td>
-      {{coord_4326|ngeoNumberCoordinates:2:{y} {x}}}
+      {{coord_4326|ngeoNumberCoordinates:2:'{y} {x}'}}
     </td>
   </tr>
   <tr>
@@ -20,7 +20,7 @@
       Wgs Coord. DMS
     </td>
     <td>
-      {{coord_4326|ngeoDMSCoordinates:0;{y} {x}}}
+      {{coord_4326|ngeoDMSCoordinates:0:'{y} {x}'}}
     </td>
   </tr>
   <tr>

--- a/contribs/gmf/examples/partials/contextualdata.html
+++ b/contribs/gmf/examples/partials/contextualdata.html
@@ -12,7 +12,7 @@
       Wgs Coord.
     </td>
     <td>
-      {{coord_4326|ngeoNumberCoordinates:2:{y} {x}}}
+      {{coord_4326|ngeoNumberCoordinates:2:'{y} {x}'}}
     </td>
   </tr>
   <tr>

